packages/web-components/src/components/cbp-usa-banner/cbp-usa-banner.stories.tsx

```diff
@@ -0,0 +1,24 @@
+export default {
+    title: 'Components/USA Banner',
+    tags: ['autodocs'],
+    argTypes: {
+      // context : {
```

Comment:
go ahead and remove this stuff. No props, so no controls needed.

Reply:
removed